config/connect-distributed.properties

```diff
@@ -39,4 +39,5 @@ internal.value.converter.schemas.enable=false
 offset.storage.topic=connect-offsets
 # Flush much faster than normal, which is useful for testing/debugging
 offset.flush.interval.ms=10000
-config.storage.topic=connect-configs
\ No newline at end of file
+config.storage.topic=connect-configs
+status.storage.topic=connect-status
```

Comment:
Ack. I think it makes sense to wait for the CreateTopic API before working on this, but we can create the JIRA now.